packages/ui-components/src/Avatar/Avatar.module.scss

```diff
@@ -0,0 +1,98 @@
+@import "../styles/tokens.scss";
+
+@mixin selectable-border($n) {
+  box-sizing: border-box;
+  border: 2px solid $n;
+}
+
+.avatar {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  font-family: Source Sans Pro, sans-serif; // TODO (koorosh): replace with fonts from tokes
+  font-style: normal;
+  font-weight: 600;
+  line-height: 1;
+  border-radius: 50%;
+  overflow: hidden;
+  user-select: none;
+}
+
+.intent-default {
+  background: $crl-neutral-2;
+  color: $crl-neutral-7;
+}
+
+.intent-info {
+  background: $crl-blue-1;
+  color: $crl-blue-3;
+}
+
+.intent-warning {
+  background: $crl-yellow-1;
+  color: $crl-yellow-4;
+}
+
+.intent-danger {
+  background: $crl-red-1;
+  color: $crl-red-4;
```

Comment:
Our warning color is `$crl-base-red`/`$crl-red-3`; comment re: base colors from above applies here too.

Reply:
> Just the one issue re: orange vs. yellow and updating intent labels but otherwise looks great
Updated PR with this change as well! Thank you for the comment!